packages/circuit-ui/components/Spinner/Spinner.spec.tsx

```diff
@@ -28,6 +28,21 @@ describe('Spinner', () => {
     expect(actual).toMatchSnapshot();
   });
 
+  it('should render a byte spinner', () => {
+    const actual = create(<Spinner size="byte" />);
+    expect(actual).toMatchSnapshot();
+  });
+
+  it('should render a kilo spinner', () => {
+    const actual = create(<Spinner size="kilo" />);
+    expect(actual).toMatchSnapshot();
+  });
+
+  it('should render a giga spinner', () => {
+    const actual = create(<Spinner size="giga" />);
+    expect(actual).toMatchSnapshot();
+  });
```

Comment:
You can avoid repetition here with [`it.each`](https://jestjs.io/docs/api#testeachtablename-fn-timeout):

```suggestion
  const sizes: SpinnerProps['size'][] = ['byte', 'kilo', 'giga'];

  it.each(sizes)('should render a %s spinner', (size) => {
    const actual = create(<Spinner size={size} />);
    expect(actual).toMatchSnapshot();
  });
```

This has the added benefit that TypeScript will alert you if the size options change and the test hasn't been updated.